modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -47,6 +47,15 @@ $ oc patch argocd openshift-gitops \
 -n openshift-gitops --type=merge \ 
 --patch-file out/argocd/deployment/argocd-openshift-gitops-patch.json
 ----
++
+[NOTE]
+====
+For a disconnected environment, you must patch the `out/argocd/deployment/argocd-openshift-gitops-patch.json` file with the `ztp-site-generator` image mirrored in your local registry by running the following command:
+[source,terminal]
+----
+$ oc patch argocd openshift-gitops -n openshift-gitops --type='json' -p='[{"op": "replace", "path": "/spec/repo/initContainers/0/image", "value": "${LOCAL_REGISTRY}/${ZTP_SITE_GENERATOR_IMAGE_REF}"}]'
```

Comment:
What should be value of  $LOCAL_REGISTRY and ${ZTP_SITE_GENERATOR_IMAGE_REF}

Please give sample examples for end-user reference.